openwisp_monitoring/device/settings.py

```diff
@@ -58,3 +58,9 @@ def get_health_status_labels():
 MAC_VENDOR_DETECTION = getattr(
     settings, 'OPENWISP_MONITORING_MAC_VENDOR_DETECTION', True
 )
+
+DEVICE_RESOURCES_THRESHOLDS = getattr(
+    settings,
+    'OPENWISP_MONITORING_DEVICE_RESOURCES_THRESHOLDS',
+    {'memory': 95, 'load': 90, 'disk_usage': 80},
```

Comment:
Sorry I wasn't clear enough. IMO it's better having the values as 0.95, 0.9, 0.8

Reply:
It can't be, they need to be integers as I realized a bit late that threshold can only be an integer.

Reply:
I see now, then OK, leave it like that and we can discuss about changing it later